src/storage/postgres/src/main/java/org/locationtech/geogig/storage/postgresql/DataSourceManager.java

```diff
@@ -64,6 +160,8 @@ protected DataSource connect(Environment.ConnectionConfig config) {
         hc.setPassword(config.getPassword());
         hc.setConnectionTimeout(5000);
 
+        String jdbcUrl = config.getServer() + ":" + config.getPortNumber() + "/"
```

Comment:
Since `jdbcUrl` is only used for the Debug log on the next line, it would be better to put this code, along with the debug log, into an `if` block that checks the log level. That would avoid the String concatenation stuff if the log wouldn't be generated anyway.